mlflow/java/client/src/main/java/org/mlflow/tracking/MlflowClient.java

```diff
@@ -658,9 +658,10 @@ private ArtifactRepository getArtifactRepository(String runId) {
    * The current available stages are: [None, Staging, Production, Archived].
    *
    *    <pre>
-   *        List<ModelVersionDetailed> detailsList = getLatestVersions("model");
+   *        import org.mlflow.api.proto.ModelRegistry.ModelVersion;
+   *        List{@code <ModelVersion>} detailsList = getLatestVersions("model");
```

Comment:
Properly escape the `<ModelVersionDetailed>` tag. Also update the type to `ModelVersion`, as that's what the API actually returns